doc/going-further/extending-castor/events.md

```diff
@@ -39,3 +39,15 @@ Here is the built-in events triggered by Castor:
 
 * `Castor\Event\AfterExecuteTaskEvent`: This event is triggered after executing
   a task. It provides access to the `TaskCommand` instance.
+
+* `Castor\Event\ProcessStartEvent`: This event is triggered after a process has
+  been started. It provides access to the `Process` instance and the current `Command`.
+
+* `Castor\Event\ProcessTerminateEvent`: This event is triggered after a process has
+  been terminated. It provides access to the `Process` instance and the current `Command`.
```

Comment:
I would also precise that process are created through the `run()` function (i think not everyone will make the link)